sklearn/tests/test_random_projection.py

```diff
@@ -64,14 +64,6 @@ def densify(matrix):
 ###############################################################################
 
 
-@pytest.mark.parametrize(
-    "n_samples, eps", [(100, 1.1), (100, 0.0), (100, -0.1), (0, 0.5)]
-)
-def test_invalid_jl_domain(n_samples, eps):
-    with pytest.raises(ValueError):
-        johnson_lindenstrauss_min_dim(n_samples, eps=eps)
-
-
```

Comment:
We also to need to keep this, but we should change the parametrization to use lists.